assets/media/alt-tag-enhancements.js

```diff
@@ -0,0 +1,89 @@
+import $ from 'jquery'
+import forEach from 'lodash/forEach'
+import get from 'lodash/get'
+import './alt-tag.scss'
+const MEDIA_IMAGE_TYPE = 'image';
```

Comment:
space between imports and const, you might want to add a prettier config file for this